src/Components/Body/Body.tsx

```diff
@@ -2,20 +2,20 @@ import {GenreSelector} from './GenreSelector/GenreSelector';
 import {SortPanel} from './SortPanel/SortPanel';
 import {MoviesList} from './MoviesList/MoviesList';
 import {ErrorBoundary} from './ErrorBoundary/ErrorBoundary';
-import './Body.scss';
+import styles from './Body.module.scss';
```

Comment:
Just note. It would be more comfortable if you use some imports order. A lot of teams preferred to import style in the last order.